README.md

```diff
@@ -54,7 +54,7 @@ rails generate devise_security:install
 ```
 
 The generator adds optional configurations to
-`config/initializers/devise-security.rb`. Enable the modules you wish to use in
```

Comment:
Per my other comment, let's add a note here that `devise-security` is supported but doesn't match Rubocop's style guide, with a link to https://docs.rubocop.org/rubocop/cops_naming.html#namingfilename